app/models/data_science/click_history.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: ds_click_histories
+#
+#  id                  :bigint           not null, primary key
+#  w2_logout_add_later :datetime
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#  client_id           :bigint           not null
+#
+# Indexes
+#
+#  index_ds_click_histories_on_client_id  (client_id) UNIQUE
+#
+# Foreign Keys
+#
+#  fk_rails_...  (client_id => clients.id)
+#
+module DataScience
+  class ClickHistory < ApplicationRecord
```

Comment:
ngl kinda hate this new model name. why are we giving data science their own namespace??

Reply:
@jenny-heath I think we should change it; I can take that on.

Although @wschaefer and I came up with the name last week and were fond of it, I talked with a data science team member (Chris Chan) today over Tuple, and he really did not like a namespace for the data science team. Chris said it's better to call this Ruby module 'analytics' or 'program' or something not targeted at the data science team. I think analytics is fine, even though there would be 3 things called analytics (AcceptedTaxReturnAnalytics, this model, and the `analytics.*` SQL VIEW).

There's some enthusiasm for having this table store _events_ (one event per row) rather than one row per client plus one column per event type. @tgrathwell suggested that to me earlier, and so did Chris Chan. For the table/class name, Chris liked the phrase "event stream." I think was Travis's idea, too.

I'll probably go with `Analytics::EventStream` or something like that and do a rename tomorrow so it's renamed before it has prod users.